static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -27,7 +28,8 @@ export const concurrentTests: $ReadOnlyArray<ABTest> = [
     commercialGptLazyLoad,
 ];
 
-export const priorityEpicTest: EpicABTest = remoteRenderEpic;
```

Comment:
As we wouldn't be running this test alongside the current one, it's easier to maintain these if we only import one or the other.